### ISSUE TITLE
Potential community confusion with Homebrew package manager (https://brew.sh aka linuxbrew)

### ISSUE THREAD

**Issue by markstinson** [SteamDeckHomebrew/PluginLoader#90]
First, this project fills a much needed gap that Valve has overlooked. ++

If you aren't aware, there's a potential for SteamDeck community confusion with Homebrew package manager (https://brew.sh aka linuxbrew)

One challenge is this project being distinguished. When folks refer to or search for "steam deck homebrew" the results are blended. Not so bad folks cannot tell a "Package Manager" from "Plugin Loader".

Another is regarding directory names. Although this project and "linuxbrew" do not use the same directory names, it may cause confusion for some less knowledgeable users. 

For awareness, given the design of Steam Decks' rootfs being read-only, many users of Desktop mode are finding installing the Homebrew package manager (https://brew.sh aka linuxbrew) to fill their CLI gaps - much like Flatpak for their GUI apps.

By default, `brew` recommends installing with sudo to /home/linuxbrew/.linuxbrew - which deck users should. However folks may also install it as /home/deck/.linuxbrew/

I'm by no means recommending you change the project or its directory namespace. I'm trying to raise awareness further keywords, tags, etc. may be required to highlight the users this is a [cool & much needed] Steam Deck Game Mode Plug-in loader/manager and not the Homebrew/Linuxbrew for Desktop Mode CLI scripts and such. Maybe even put a notice of redirect in the project readme, "for Homebrew/Linuxbrew CLI - see https://brew.sh"

Thanks again for the awesome Steam Deck enhancement. Cheers.

https://en.wikipedia.org/wiki/Homebrew_(package_manager) - since 2009
https://github.com/Homebrew/brew - 255 releases
https://docs.brew.sh/Homebrew-on-Linux
https://gist.github.com/uyjulian/105397c59e95f79f488297bb08c39146 - howto for Steam Deck
https://brew.sh

**Comment by markstinson** [comment]
Facepalm ... Thanks for the reminder. I've been so focused on my non-EMU, Steam games Deck-wise, I completely spaced on that. I've set down all my other personal projects & games given how engrossing & flexible Steam Deck is - my hyperfocus is a feature (and flaw).

When mentioning you didn't think folks would use "brew" on the Steam Deck, I'm curious as to what context, namespace, or item you were referencing. I ask so don't make the same mistake again.

When I think "brew", I think linuxbrew (also a Homebrew) given its popularity with Konsole users filling a much needed gap. It has the command `brew` (and namespace `linuxbrew`) and usage will grow as more units ship.

Thanks again for such a boss util.

**Comment by JakobDev** [comment]
> When mentioning you didn't think folks would use "brew" on the Steam Deck, I'm curious as to what context, namespace, or item you were referencing. I ask so don't make the same mistake again.

Just a thought of me. Most people will use the Deck as Gaming Console and not use the Terminal. I also use Linux for years an be part of the Linux community  and I have nobody ever seen mentioning Brew . I heard from Brew only in the context of Mac OS. From the Webpage, I don't see any things that a normal Linux package Manager e.g. pacman does not offer.

**Comment by markstinson** [comment]
linuxbrew doesn't offer anything too terribly different than what's installable via pacman - probably less. `brew` does fill a use-case -- it was designed to bring linux/gnu commands to user-owned space without interfering with system installs of the same name and/or for users lacking elevated rights for installs.

On the Steam Deck, I can enable r/w and do pacman installs -- only to lose them then needing to re-install on the next OS update -- which I've had to do several times. Fed up, I installed linuxbrew. ATM, I'm working on my Steam Deck ansible playbooks (with a howto for a virtualenv of ansible-core) given my flatpak re-installs due to resets. I've other playbook ideas I'll be kicking around: installing this project, freeing space, brew & pacman package management, etc.

A `brew` install is most definitely based on use-case, just like folks needing to enable 'r/w' for pacman on this handheld pc console. `brew` is definitely a middleground for newer folks hesitant to use r/w on the rootfs. As soon as Valve slows down some of their updates, I plan on returning to using pacman again myself as `brew` can waste gigs of space.
